src/platform/testing/visual-regression/util/get-file-names.js

```diff
@@ -38,7 +38,7 @@ function getFileNames(route) {
 async function createDirectoryIfNotExist(filePath) {
   const directory = path.dirname(filePath);
   return new Promise((resolve, reject) => {
-    mkdirp(directory, err => (err ? reject(err) : resolve()));
+    fs.ensureDir(directory, err => (err ? reject(err) : resolve()));
```

Comment:
`fs-extra` offers what we were using `mkdirp` for, so I swapped out the module here and uninstalled it.

CC: @RobAWilkinson